interactors/generic.js

```diff
@@ -0,0 +1,10 @@
+import { createInteractor, focused } from '@bigtest/interactor';
```

Comment:
This should be called `Element`

Reply:
I was thinking in terms of the built-in elements, and I was a worried about confusion with the HTMLElement. Maybe not an issue?

Reply:
Ah yeah, is there a global Element constructor? I think it’s fine to shadow it although it might screw up typescript.  🤔

Reply:
I switched it to `HTML`. The other contender was `Any`, but I think we were leaning towards `HTML` since it is a little more descriptive.